jwst/datamodels/schemas/mask.schema.yaml

```diff
@@ -1,5 +1,7 @@
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: subarray.schema.yaml
+- $ref: keyword_preadpatt.schema.yaml
```

Comment:
MASK files use DETECTOR, SUBARRAY, and READPATT (in an or-like fashion) as CRDS selectors, so this looks good. I'm guessing the dq_init step code does not use any other keywords from the reference file.